### PR TITLE
Add the PointerEvent api

### PR DESCRIPTION
**Pull request by lpd-au**

### CODE REVIEW

File: api/PointerEvent.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+              }
+            }
+          ],
+          "ie": [
+            {
+              "version_added": "11"
+            },
+            {
+              "version_added": "10",
+              "prefix": "MS",
```
Comment (teoli2003) [inline api/PointerEvent.json]: Lower case "ms". It is not case-sensitive and it is our convention (We wouldn't use MOZ or WEBKIT, but the lower case variants)

File: api/PointerEvent.json [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+            "version_added": "55"
+          },
+          "chrome": {
+            "version_added": "55"
+          },
+          "chrome_android": {
+            "version_added": "55"
+          },
+          "edge": {
+            "version_added": true
```
Comment (teoli2003) [inline api/PointerEvent.json]: "12"

File: api/PointerEvent.json [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+          },
+          "edge_mobile": {
+            "version_added": true
```
Comment (teoli2003) [inline api/PointerEvent.json]: "12"

File: api/PointerEvent.json [hunk 4]
```diff
@@ -0,0 +1,212 @@
...
+              "notes": "See <a href='https://msdn.microsoft.com/library/dn304886'>MSDN Pointer events updates</a>."
+            }
+          ],
+          "ie_mobile": [
+            {
+              "version_added": "11"
+            },
+            {
+              "version_added": "10",
+              "prefix": "MS",
```
Comment (teoli2003) [inline api/PointerEvent.json]: Lower case.

File: api/PointerEvent.json [hunk 5]
```diff
@@ -0,0 +1,212 @@
...
+            "version_added": false
+          }
+        },
+        "status": {
+          "experimental": false,
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "level-two-pen-attributes": {
```
Comment (teoli2003) [inline api/PointerEvent.json]: Separate in two entries, one for each property.

File: api/PointerEvent.json (hunk 1 above)
Comment (lpd-au) [reply api/PointerEvent.json]: I took it from here: https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix
Is that out of date? It specifically instructs uppercase for interfaces.

Comment (teoli2003) [reply api/PointerEvent.json]: Good point. Forget about my lowercase comments then.

**Comment by lpd-au** [comment]
@teoli2003 Do I also need to include the various related pointerCapture properties and methods that are on Element, not PointerEvent?

**Comment by teoli2003** [comment]
Nope, only the ones on PointerEvent. The ones on Element (via the GlobalEventHandlers mixin or other). Will be added in an element.json when we do this one (yes it will be a complex one :-) )

**Comment by lpd-au** [comment]
Ok, well, I think this should be finished then :)

**Pull request merged**